features/zoeken.feature

```diff
@@ -0,0 +1,53 @@
+# language: nl
+
+Feature: zoeken van WOZ-objecten
+  WOZ-objecten zoeken op kadastraal onroerende zaak identificatie, waarbij ook WOZ-objecten worden gevonden waarbij niet de kadastraal onroerende zaak identificatie, maar we de kadastrale aanduiding bekend is.
```

Comment:
de zin na de eerste comma is niet duidelijk. Wordt hier bedoeld


waarbij ook WOZ-objecten worden gevonden **waarvan** niet de kadastraal onroerende zaak identificatie, maar we**l** de kadastrale aanduiding bekend is.